src/common/components/header/index.js

```diff
@@ -18,18 +18,18 @@ class Header extends Component {
           { authenticated
             ?
               <div className={ styles.sideNav }>
-                { user && <a href={user.html_url} className={ styles.link }>{user.name}</a> }
+                { user && Hi, {user.name} }
```

Comment:
This wouldn't work this way - we still need some wrapping element to properly render and style it. 

The simplest way to fix it would be to make it:
`{ user && <span className={ styles.link } >Hi, {user.name}</span> }`

This will simply apply the same style to it as to links (in terms of positioning in header) - it will not make it clickable or change on hover.